oak-jcr/src/test/java/org/apache/jackrabbit/oak/jcr/security/authorization/VersionManagementTest.java

```diff
@@ -45,7 +45,7 @@ public class VersionManagementTest extends AbstractEvaluationTest {
 
     @Override
     @Before
-    protected void setUp() throws Exception {
+    public void setUp() throws Exception {
```

Comment:
why are changes needed in this class?

Reply:
I reverted all of the changes, since the new unit tests are now in a different class.